src/app/contextBroker/contextBroker.cpp

```diff
@@ -783,495 +783,393 @@ static const char* validLogLevels[] =
 
 
 
-#define API_V2                                                                                         \
```

Comment:
Yes, the PR is all about removing the VERB/METHOD from the vector and splitting the ONE vector into X vectors, one per verb (the last one is special, the one I call *badVerbV*, containing the service routines where no adequate verb/method has been found)

Reply:
Sure, I could move the vector to another CPP file (and its header), but ... what do we gain with that?
Just more complexity, in my opinion

Reply:
I think is more clear to have all routes in the same place, not mixed with the rest of logic of the contextBroker.cpp (initialization functions, etc.).

Reply:
Mixed?
Not sure what you mean with that ...
The service routine vectors are where they have always been, right where they should be and right where they are used.
It's been like this since 2011, so ... very hard to understand why we should move them out to a different file (where they are not used) now ...
The vectors are needed in the call to restInit(), that will stay inside contextBroker.cpp.
I see no reason whatsoever to create another file only with the service vectors.

Reply:
I wouldn't use the "it has been that way from a time long" argument because if we follow it, then even the refactor done by this PR could be put on doubt :)

Routes configuration is pretty much static and could be considered part of the rest library, so the movement could make sense, changing the way restInit() works, i.e. restInit() would pass the vector as arguments (as much it would select the mode to use, e.g. cors, no-cors, etc.).

However, it is not so important at the present moment, so we can keep it as it is now and think again in the future.

NTC

Reply:
If we do the refactoring mentioned in this PR, about having restInit less "flexible", which we needed yeara ago but no longer, then the service vectors would go into the rest library, no doubt.
It's not a bad idea, let's do it. But, in a different PR ...

Reply:
Maybe adding `// FIXME PR: to be removed within #3109 refactor task` would be useful. I think I will not forget about this but... never knows :)

Reply:
Fixed in 74c7a9c